helix_fhir_client_sdk/fhir_client.py

```diff
@@ -2715,3 +2734,17 @@ async def simulate_graph_async(
             logger=self._logger,
             auth_scopes=self._auth_scopes,
         )
+
+    def add_additional_request_headers(self, headers: Dict[str, str]) -> None:
+        """
+        Add additional request headers while making any request
+        """
+        additional_headers: Dict[str, str] = self._additional_request_headers
+        if additional_headers:
+            if "User-Agent" in self._additional_request_headers.keys():
+                self._additional_request_headers["User-Agent"] = (
+                    f"Python/{sys.version_info.major}.{sys.version_info.minor}; "
```

Comment:
Is python, aiohttp version required? Shouldn't additional header have (pipeline name or loader name with team name) because then person receiving request can easily ping the requested team to look at the request their sending.